File: DKPython/functions/dk_success.py
import sys 
sys.path.append(".")
import DK

from dk_log import *

##################################################################################
# dk_success(<message>)
#
#    Print a success message to the console
#
#    @message - The message to print
#
def dk_success(*args):
#    dk_call dk_debugFunc 1

    dk_log(SUCCESS, args[0])





###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
def DKTEST():
#    dk_call dk_debugFunc 0

    dk_success("test dk_success message")

File: DKPhp/functions/dk_success.php
#!/usr/bin/php
<?php

################################################################################
# dk_success(message)
#
#    Print a success message to the console
#
#    @message - The message to print
#
function dk_success() {
	#dk_debugFunc(1);
	dk_source("dk_log");
	
	if(!isset($argv)){
		$argv = func_get_args();
	}
	
    dk_log("SUCCESS", $argv[0]);
}




###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
if(!function_exists('DKTEST')){ function DKTEST() {
	#dk_debugFunc 0

    dk_success("test dk_success message");
}}


include_once(str_replace("\\","/",$_SERVER['USERPROFILE'])."/Digital Knob/Development/DKPhp/functions/DK.php");
?>